R/object_name_linter.R

```diff
@@ -44,6 +44,11 @@ object_name_xpath <- local({
 #' @param styles A subset of
 #'   \Sexpr[stage=render, results=rd]{lintr:::regexes_rd}. A name should
```

Comment:
aside: could this be substituted with `@evalRd`? do I understand what this code is doing correctly? (neat trick btw, never seen it before...)

Reply:
IIRC `@evalRd` doesn't work "within" other roxygen tags. The trick goes back to Jim, though. Maybe he has more insight.

Reply:
OK, let's punt for now